services/core-web/common/utils/helpers.js

```diff
@@ -227,3 +228,28 @@ export const formatComplianceCodeValueOrLabel = (code, showDescription) => {
 
   return `${section}${formattedSubSection}${formattedParagraph}${formattedSubParagraph}${formattedDescription}`;
 };
+
+// function to flatten an object for nested items in redux form
+// eslint-disable-snippets
+export const flattenObject = (ob) => {
+  const toReturn = {};
+  let flatObject;
+  for (const i in ob) {
+    if (!ob.hasOwnProperty(i)) {
```

Comment:
shouldn't this always return false since your iterating through the values on the object?

Reply:
yes you are correct, I'll remove this block on both files.

Reply:
this block has been removed